README.md

```diff
@@ -63,6 +63,19 @@ urlpatterns = [
     )
 ```
 
+#### Topic pages
+- Topic pages are optional as well and can be enabled by using the view `canonicalwebteam.blog.django.views.topic`. The view takes the topic slug to fetch data for and a template path to load the correct template from.
```

Comment:
I'm not sure we should do topics this way. Unlike with Categories, where it was top-level URLs that couldn't be provided generically (`blog/cloud-and-server`), these URLs are easily generalisable (`blog/topics/{topic_name}`).

So in this case I think maybe we should just make this a setting that you add to `webapp/settings.py`, like `ENABLE_TOPIC_PAGES: True`. What do you think?

Reply:
Sounds good. The template will then correspond to the slug. Makes it generic enough

Reply:
huh? you need a different template per topic page? Maybe that's why you did it this way?

Reply:
In that case, maybe your solution is better

Reply:
Updated it :)

Reply:
Thanks, sorry